changelog/15614.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+elasticsearch: Use the new /_security base API path instead of /_xpack/security when managing elasticsearch.
```

Comment:
I think the format is usually a little more like this:

```suggestion
secrets/database/elasticsearch: Use the new /_security base API path instead of /_xpack/security when managing elasticsearch.
```

Example: https://github.com/hashicorp/vault/blame/main/CHANGELOG.md#L1034